src/utils/imageLargeView/imageLargeView.js

```diff
@@ -0,0 +1,34 @@
+import Vue from 'vue';
+
+import ImageLargeView from './imageLargeView.html';
+
+export function openInImageLargeView(openmct, imageSrc, imageMeta) {
+    const DEFAULT_FILTERS = {
+        brightness: 100,
+        contrast: 100
+    };
+
+    const alt = imageMeta.alt || 'missing image';
```

Comment:
You may want to check for imageMeta before assigning to alt, just like filters below it, if no meta is passed in it'll throw that "Cannot read property 'alt' of undefined" error.

Reply:
Oh and why "missing image" for the default?

Reply:
Maybe defaulting imageMeta = {} in the params will negate all that.

Reply:
added default value